daft/expressions/expressions.py

```diff
@@ -1489,7 +1973,23 @@ def find(self, substr: str | Expression) -> Expression:
             If the substring is not found, -1 is returned.
 
         Example:
-            >>> col("x").str.find("foo")
+            >>> import daft
+            >>> df = daft.from_pydict({"x": ["foo", "bar", "baz"]})
```

Comment:
Let's have a more motivating example? You can have a entry like `foobar` that returns 0, 'barfoo' that returns 3, etc.